challenge.rb

```diff
@@ -1,26 +1,26 @@
 
 def capitalize_each_string(input)
-  #implement your solution here
+    return input.map {|item| item.capitalize}
```

Comment:
Rubyists tend to prefer implicitly returning the final value of a method. https://github.com/bbatsov/ruby-style-guide#no-explicit-return